{0} setUpClass (tempest.api.volume.admin.test_encrypted_volumes_extend.EncryptedVolumesExtendAttachedTest) ... SKIPPED: EncryptedVolumesExtendAttachedTest skipped as Neutron is not available
{2} setUpClass (tempest.api.compute.volumes.test_attach_volume.AttachVolumeMultiAttachTest) ... SKIPPED: Neutron is not available
{1} tempest.api.compute.volumes.test_volume_snapshots.VolumesSnapshotsTestJSON.test_volume_snapshot_create_get_list_delete [6.134730s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list [0.104960s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_limit [0.074509s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_offset_and_limit [0.146079s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_limit [0.071267s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_offset_and_limit [0.136064s] ... ok
{2} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_details [0.068293s] ... ok
{1} tempest.api.compute.volumes.test_volumes_get.VolumesGetTestJSON.test_volume_create_get_delete [2.886625s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_create_group_from_group_snapshot [13.244200s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.100286s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.013932s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.012527s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.038205s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.009741s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.009000s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.035246s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.041108s] ... ok
{2} tempest.api.volume.admin.test_group_type_specs.GroupTypeSpecsTest.test_group_type_specs_create_show_update_list_delete [0.572498s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_compare_volume_stats_values [1.079371s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_get_capabilities_backend [0.539883s] ... ok
{1} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_create_list_update_show_delete [0.396749s] ... ok
{1} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_list_by_optional_params [0.291488s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_multi_backend.VolumeMultiBackendTest) ... SKIPPED: Cinder multi-backend feature disabled
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_create_show_list_delete [8.682048s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_delete_group_snapshots_following_updated_volumes [17.423807s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_associate_disassociate_qos [0.729738s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_back_end_consumer [0.199143s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_both_consumer [0.178934s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_front_end_consumer [0.168751s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_get_qos [0.020501s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_list_qos [0.023005s] ... ok
{1} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_set_unset_qos_key [0.140214s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_group_snapshot_create_show_list_delete [9.071302s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_update [10.666789s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_reset_snapshot_status [0.291981s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_creating [3.852270s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_deleting [3.554002s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status [9.518971s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_snapshot_manage.SnapshotManageAdminTest) ... SKIPPED: SnapshotManageAdminTest skipped as Neutron is not available
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error [3.587424s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsV314Test.test_create_group_from_group [11.762054s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error_deleting [3.597138s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_update_snapshot_status [0.237326s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_delete_message [2.265978s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_list_show_messages [1.965169s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_volume_manage.VolumeManageAdminTest) ... SKIPPED: Manage volume tests are disabled
{2} tempest.api.volume.admin.test_groups.GroupsV320Test.test_reset_group_status [3.320141s] ... ok
{1} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_list_hosts [0.129001s] ... ok
{1} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_show_host [0.068829s] ... ok
{0} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_show_default_quota [0.733047s] ... ok
{0} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_update_default_quota [1.716633s] ... ok
{1} tempest.api.volume.admin.test_volume_pools.VolumePoolsAdminTestsJSON.test_get_pools_without_details [0.666668s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volume_gigabytes [0.481757s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volumes [0.470050s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_volume_extend_gigabytes_quota_deviation [0.390407s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_create_get_delete [0.327613s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_encryption_create_get_update_delete [0.397598s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_list [0.037112s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_update [0.423480s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_delete_quota [0.777113s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_default_quotas [0.136511s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_quotas [0.150487s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_spec_create_get_delete [0.157831s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_list [0.078541s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_update [0.207314s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage [4.150196s] ... ok
{2} tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithoutMigrationTest.test_available_volume_retype [5.347286s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_invalid_body [0.021527s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_none_body [0.019518s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_nonexistent_type_id [0.038855s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_delete_nonexistent_volume_type_id [0.021446s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_extra_spec_name [0.027356s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_volume_type_id [0.023496s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_list_nonexistent_volume_type_id [0.021773s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_multiple_extra_spec [0.015654s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_no_body [0.017738s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_none_extra_spec_id [0.031654s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_extra_spec_id [0.040599s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_type_id [0.042748s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage_after_volume_transfer [5.949208s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_show_quota_usage [0.177850s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_update_all_quota_resources_for_tenant [0.632664s] ... ok
{0} tempest.api.volume.test_availability_zone.AvailabilityZoneTestJSON.test_get_availability_zone_list [0.187747s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_host_name [0.027805s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_and_host_name [0.026996s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_binary_name [0.023921s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithMigrationTest) ... SKIPPED: Cinder multi-backend feature disabled
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_volume_host_name [3.962653s] ... ok
{2} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_list_services [0.026069s] ... ok
{0} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_crud_snapshot_metadata [0.257623s] ... ok
{1} tempest.api.volume.admin.test_volume_type_access.VolumeTypesAccessTest.test_volume_type_access_list [0.447792s] ... ok
{0} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_update_show_snapshot_metadata_item [0.263159s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_log_reason_with_no_reason [0.019776s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_service_with_invalid_binary [0.029767s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_enable_service_with_invalid_host [0.024411s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_freeze_host_with_invalid_host [0.020086s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_thaw_host_with_invalid_host [0.041731s] ... ok
{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_crud_volume_metadata [0.360714s] ... ok
{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_update_show_volume_metadata_item [0.221907s] ... ok
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_gigabytes_snapshots [0.638128s] ... ok
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_snapshots [0.146613s] ... ok
{1} tempest.api.volume.admin.test_volumes_list.VolumesListAdminTestJSON.test_volume_list_param_tenant [2.901703s] ... ok
{2} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_volume_type_encryption_nonexistent_type_id [0.169524s] ... ok
{2} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_empty_name [0.015802s] ... ok
{2} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_repeated_name [0.152570s] ... ok
{2} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_delete_nonexistent_type_id [0.021411s] ... ok
{2} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_get_nonexistent_type_id [0.017650s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_get_list_accept_volume_transfer [5.879842s] ... ok
{2} setUpClass (tempest.api.volume.admin.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{1} setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata) [0.000000s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}

{2} tempest.api.volume.test_extensions.ExtensionsTestJSON.test_list_extensions [0.197930s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_list_delete_volume_transfer [3.972799s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_list_versions [0.012788s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_show_version [0.078052s] ... ok
{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume [0.147962s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:24:23,695 102187 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.145s
2025-03-01 10:24:23,695 102187 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-9694f974-7ba9-426d-84ef-eb4914e38607', 'date': 'Sat, 01 Mar 2025 10:24:23 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{1} tempest.api.volume.test_volume_absolute_limits.AbsoluteLimitsTests.test_get_volume_absolute_limits [0.510050s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend [0.185200s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:24:30,047 102189 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.183s
2025-03-01 10:24:30,047 102189 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-06d31f37-58f0-447c-830e-03ff39cb3b72', 'date': 'Sat, 01 Mar 2025 10:24:30 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_volume [8.590694s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_delete_cascade [8.620018s] ... ok
{1} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend_when_volume_has_snapshot [8.577286s] ... ok
{0} tempest.api.volume.test_volumes_clone_negative.VolumesCloneNegativeTest.test_create_from_volume_decreasing_size [4.311823s] ... ok
{0} setUpClass (tempest.api.volume.test_volumes_extend.VolumesExtendAttachedTest) ... SKIPPED: VolumesExtendAttachedTest skipped as Neutron is not available
{0} tempest.api.volume.test_volumes_get.VolumesSummaryTest.test_show_volume_summary [0.142933s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete [9.957922s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete [7.833343s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list [0.041628s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_by_name [0.037446s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_by_name [0.058294s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_pagination [0.259822s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_with_multiple_params [0.122723s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_pagination [0.179767s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_param_display_name_and_status [0.036095s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_display_name_and_status [0.059191s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_marker [0.070514s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_metadata [0.081265s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_details [0.064537s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_param_metadata [0.034438s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_availability_zone [0.063540s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_bootable [0.072120s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_status [0.078625s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_availability_zone [0.064273s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_bootable [0.064741s] ... ok
{0} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_status [0.063978s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_get_list_accept_volume_transfer [5.625959s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_list_delete_volume_transfer [4.091657s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_as_clone [11.381466s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image [0.022223s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:25:00,959 102189 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-03-01 10:25:00,959 102189 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-7f5d2daa-dc02-4120-bd1b-e473309a18d9', 'date': 'Sat, 01 Mar 2025 10:25:00 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{1} setUpClass (tempest.api.volume.test_volumes_snapshots.VolumesSnapshotTestJSON) ... SKIPPED: VolumesSnapshotTestJSON skipped as Neutron is not available
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_with_nonexistent_volume_id [0.263109s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_without_passing_volume_id [0.029313s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshot_invalid_param_limit [0.016347s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_marker [0.024823s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_sort [0.017136s] ... ok
{1} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_volume_from_snapshot_decreasing_size [7.281810s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_get_list_accept_volume_transfer [5.920799s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_list_delete_volume_transfer [4.000032s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_deactivated_image [0.532945s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_image_with_decreasing_size [0.366628s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.023256s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_snapshot_id [0.027118s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_source_volid [0.028757s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_volume_type [0.031929s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_negative [0.016262s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.016927s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.016253s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.027295s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.011127s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_detach_volumes_with_invalid_volume_id [0.033021s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_invalid_volume_id [0.028289s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.011815s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_invalid_status [0.049404s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_nonexistent_name [0.050307s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_invalid_status [0.028951s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_nonexistent_name [0.034162s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_negative_volume_status [0.123630s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_nonexistent_volume_id [0.073484s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_unreserve_volume_with_nonexistent_volume_id [0.026776s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_empty_volume_id [0.011310s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_invalid_volume_id [0.028960s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_nonexistent_volume_id [0.029022s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.028773s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_None_size [0.016110s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_non_number_size [0.015535s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_nonexistent_volume_id [0.029536s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_size_smaller_than_original_size [0.015809s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_without_passing_volume_id [0.034482s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.027774s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit [0.029383s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_infinite [0.081357s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_zero [0.027268s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_marker [0.067200s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_offset [0.041900s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_asc [0.048036s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_desc [0.030630s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_asc [0.030996s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_desc [0.029724s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_asc [0.030283s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_desc [0.031830s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_details_with_params [0.091702s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_with_params [0.086223s] ... ok

==============================
Failed 4 tests - output below:
==============================

setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata)
------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume[id-cbbcd7c6-5a6c-481a-97ac-ca55ab715d16,image]
--------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:24:23,695 102187 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.145s
2025-03-01 10:24:23,695 102187 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-9694f974-7ba9-426d-84ef-eb4914e38607', 'date': 'Sat, 01 Mar 2025 10:24:23 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend[id-9a36df71-a257-43a5-9555-dc7c88e66e0e]
--------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:24:30,047 102189 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.183s
2025-03-01 10:24:30,047 102189 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-06d31f37-58f0-447c-830e-03ff39cb3b72', 'date': 'Sat, 01 Mar 2025 10:24:30 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image[id-54a01030-c7fc-447c-86ee-c1182beae638,image,smoke]
-------------------------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-01 10:25:00,959 102189 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-03-01 10:25:00,959 102189 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-7f5d2daa-dc02-4120-bd1b-e473309a18d9', 'date': 'Sat, 01 Mar 2025 10:25:00 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'



======
Totals
======
Ran: 203 tests in 208.9756 sec.
 - Passed: 189
 - Skipped: 10
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 4
Sum of execute time for each test: 252.7671 sec.

==============
Worker Balance
==============
 - Worker 0 (60 tests) => 0:02:46.213135
 - Worker 1 (55 tests) => 0:03:01.035905
 - Worker 2 (88 tests) => 0:03:28.975621
